wpilibc/src/main/native/cpp/drive/DifferentialDrive.cpp

```diff
@@ -157,8 +166,8 @@ DifferentialDrive::WheelSpeeds DifferentialDrive::TankDriveIK(
 }
 
 void DifferentialDrive::StopMotor() {
-  m_leftMotor->StopMotor();
-  m_rightMotor->StopMotor();
+  m_leftMotor(0.0);
+  m_rightMotor(0.0);
```

Comment:
Just to confirm, we're fine with this change in behavior? Notably, it will delay when the motors are actually stopped, since `Set()` calls `MotorSafety::Feed()`, and there's a few other differences between `Set(0)` and `StopMotor()` in `NidecBrushless`.

Reply:
Feeding the watchdog there would cause a 20 ms delay in most cases, which doesn't seem noticeable.

The main difference between Nidec's Set() and StopMotor() is whether the PWM line is disabled (I swear, the way that motor gets driven is _really_ weird). That motor is used by so few people, I'm personally fine ignoring the difference.

Reply:
Got it, this is good to be resolved. (Since I'm not the PR author and not a repo maintainer, I can't mark comments as resolved myself)